### PR TITLE
Better looking ghost notifications

### PR DESCRIPTION
**Pull request by jlsnow301**
## About The Pull Request
The alerts have been given some style
image

Also the screen tip should be quite obvious 
image

Cleaned up some code around the proc itself - really needed some TLC. That, and added an arg to minor announcements to override colors.
## Why It's Good For The Game
Better styling for alerts
## Changelog
:cl:
add: Just in time for Halloween- ghost notifications have been upgraded to their own announcements. Boo!
/:cl:

### CODE REVIEW

File: code/modules/mob/mob_helpers.dm [hunk 1]
```diff
@@ -258,62 +258,85 @@
...
-			continue
-		var/ui_style = ghost.client?.prefs?.read_preference(/datum/preference/choiced/ui_style)
-		if(ui_style)
-			alert.icon = ui_style2icon(ui_style)
-		if (header)
-			alert.name = header
-		alert.desc = message
-		alert.action = action
-		alert.target = source
-		if(!alert_overlay)
```
Comment (jlsnow301) [inline code/modules/mob/mob_helpers.dm:291]: There has to be another way to do all of this, this is all so cursed

File: code/__HELPERS/priority_announce.dm [hunk 2]
```diff
@@ -108,7 +108,7 @@
  * sound_override - optional, use the passed sound file instead of the default notice sounds.
  * should_play_sound - Whether the notice sound should be played or not.
  */
-/proc/minor_announce(message, title = "Attention:", alert = FALSE, html_encode = TRUE, list/players = null, sound_override = null, should_play_sound = TRUE)
+/proc/minor_announce(message, title = "Attention:", alert = FALSE, html_encode = TRUE, list/players, sound_override, should_play_sound = TRUE, color_override)
```
Comment (san7890) [inline code/__HELPERS/priority_announce.dm:111]: add your new argument to the documentation

File: code/modules/mob/mob_helpers.dm [hunk 3]
```diff
@@ -258,62 +258,58 @@
...
-		alert.add_overlay(alert_overlay)
 
-/**
- * Heal a robotic body part on a mob
- */
+		var/atom/movable/screen/alert/notify_action/toast = ghost.throw_alert( \
+			category = "[REF(source)]_notify_action", \
+			type = /atom/movable/screen/alert/notify_action, \
+			new_master = source, \
+		)
```
Comment (san7890) [inline code/modules/mob/mob_helpers.dm:298]:
```suggestion
		var/atom/movable/screen/alert/notify_action/toast = ghost.throw_alert(
			category = "[REF(source)]_notify_action",
			type = /atom/movable/screen/alert/notify_action,
			new_master = source,
		)
```

no need for these

File: code/modules/mob/mob_helpers.dm [hunk 4]
```diff
@@ -258,62 +258,58 @@
...
+	atom/source, \
+	mutable_appearance/alert_overlay, \
+	action = NOTIFY_JUMP, \
+	flashwindow = TRUE, \
+	ignore_mapload = TRUE, \
+	ignore_key, \
+	header, \
+	notify_suiciders = TRUE, \
+	notify_volume = 100 \
+) //Easy notification of ghosts.
```
Comment (san7890) [inline code/modules/mob/mob_helpers.dm:274]:
```suggestion
/proc/notify_ghosts(
	message,
	ghost_sound,
	enter_link,
	atom/source,
	mutable_appearance/alert_overlay,
	action = NOTIFY_JUMP,
	flashwindow = TRUE,
	ignore_mapload = TRUE,
	ignore_key,
	header,
	notify_suiciders = TRUE,
	notify_volume = 100
)
```

no need for backslashes when it's a proc, we only really need backslashes in macro-wrapped stuff like `AddComponent` and `AddElement`. BYOND-standard stuff works natively with multilining. Also that comment is not useful so let's axe it.

**Review by san7890 — CHANGES_REQUESTED** [review]

File: code/modules/mob/mob_helpers.dm (hunk 3 above)
Comment (jlsnow301) [reply code/modules/mob/mob_helpers.dm:298]: wew

**Pull request merged**